vector/src/main/res/layout/dialog_change_password.xml

```diff
@@ -1,31 +1,67 @@
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:orientation="vertical"
-    android:paddingStart="?dialogPreferredPadding"
-    android:paddingLeft="?dialogPreferredPadding"
+    android:paddingStart="?android:attr/dialogPreferredPadding"
+    android:paddingLeft="?android:attr/dialogPreferredPadding"
     android:paddingTop="12dp"
-    android:paddingEnd="?dialogPreferredPadding"
-    android:paddingRight="?dialogPreferredPadding">
+    android:paddingEnd="?android:attr/dialogPreferredPadding"
+    android:paddingRight="?android:attr/dialogPreferredPadding"
+    tools:targetApi="lollipop_mr1">
```

Comment:
why this targetApi attribute?

Reply:
Android Studio shows warning for using `?android:attr` attribute just to remove the warning from editor

Reply:
and what the warning says?

Reply:
earlier it wasn't working without `?android:attr/` thats why added it . But now its working. 
and as per warning . It was giving warning `?android:attr` requires API 22 . 
but now its resolved . removed `?android:attr/` :)